advanced/enable_extensions.md

```diff
@@ -32,13 +32,14 @@ activation commands in your user enviroment script.
 
 # Enable Jupyter extension not provided by SWAN.
 
-Install Jupyter extension requires to run the next commands from a SWAN terminal:
+Proposal: It is also possible to install and enable a Jupyter extension which is not provided by default by SWAN. The following commands show how that can be done, from the SWAN terminal:
```

Comment:
`Proposal` should not be in the text :smile:

Reply:
I removed it :)